packages/nextra-theme-docs/css/styles.css

```diff
@@ -22,19 +22,18 @@ body {
   @apply _w-full dark:_text-gray-100;
 }
 
-a,
-summary,
-button,
-input,
-[tabindex]:not([tabindex='-1']) {
+.nextra-focusable,
+.nextra-landmark a,
```

Comment:
These classnames I am not particular about; rename to anything else if they are more consistent with existing styles